atomica/migration.py

```diff
@@ -360,4 +360,16 @@ def model_tidying(proj):
         for prog in progset.programs.values():
             prog.capacity_constraint = prog.capacity
             del prog.capacity
-    return proj
\ No newline at end of file
+    return proj
+
+@migration('1.0.25', '1.0.26', 'Rename link labels')
```

Comment:
For the version conflict, I think this one might need to be changed too? To `@migration('1.0.27', '1.0.28', 'Rename link labels')`